docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1139,6 +1139,34 @@ The limit protects the system’s stability from potential abuse or mistakes, an
 
 > **Note**: Invalid metrics metadata are skipped during the ingestion, and valid metadata within the same request are ingested.
 
+### err-mimir-distributor-max-ingestion-rate
+
+This critical error occurs when the rate of received samples per second is exceeded in a distributor.
+
+The distributor implements a rate limit on the samples per second that can be ingested, and it's used to protect a distributor from overloading in case of high traffic.
+The limit is a per-instance limit and it's applied on all samples received, across all tenants, in each distributor.
+
+How to **fix** it:
+
+- Scale up distributors.
+- Increase the limit by using the `-distributor.instance-limits.max-ingestion-rate` option (or `max_ingestion_rate` in the runtime config).
```

Comment:
Contrary to ingester instance limits, distributor instance limits are not configurable in the runtime config. It means that to change these limits, you need to restart the distributor process. It's typically not a big deal for distributors because they're stateless and quick to restart (when deployed in microservices mode).

```suggestion
- Increase the limit by using the `-distributor.instance-limits.max-ingestion-rate` option.
```